server/channels/app/export_test.go

```diff
@@ -1144,6 +1144,77 @@ func TestExportDeletedTeams(t *testing.T) {
 	}
 }
 
+func TestExportSingleTeam(t *testing.T) {
+	th1 := Setup(t).InitBasic()
+	defer th1.TearDown()
+
+	teams1, errTeams := th1.App.GetAllTeams()
+	assert.Nil(t, errTeams)
+	assert.Equal(t, len(teams1), 1)
+
+	channels1, err1 := th1.App.GetAllChannels(th1.Context, 0, 10, model.ChannelSearchOpts{})
+	assert.Nil(t, err1)
```

Comment:
Lets' call `require` instead of `assert` on all error checks to avoid NPEs in tests.